File: OPENAPI-REST-API/swagger-client/dart/lib/model/home_services_licenses_links.dart
part of swagger.api;

class HomeServicesLicensesLinks {
  /* Link to a license. */
  String n386522 = null;

  HomeServicesLicensesLinks();

  @override
  String toString() {
    return 'HomeServicesLicensesLinks[n386522=$n386522, ]';
  }

  HomeServicesLicensesLinks.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    n386522 = json['386522'];
  }

  Map<String, dynamic> toJson() {
    return {
      '386522': n386522
     };
  }

  static List<HomeServicesLicensesLinks> listFromJson(List<dynamic> json) {
    return json == null ? new List<HomeServicesLicensesLinks>() : json.map((value) => new HomeServicesLicensesLinks.fromJson(value)).toList();
  }

  static Map<String, HomeServicesLicensesLinks> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, HomeServicesLicensesLinks>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new HomeServicesLicensesLinks.fromJson(value));
    }
    return map;
  }
}

File: OPENAPI-REST-API/swagger-client/java-vertx/src/main/java/io/swagger/server/api/model/HomeServicesLicensesLinks.java
package io.swagger.server.api.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import io.swagger.v3.oas.annotations.media.Schema;
import io.vertx.core.json.JsonObject;
import io.vertx.codegen.annotations.DataObject;
/**
 * Map of license service IDs to their IP addresses for the account dashboard.
 */




public class HomeServicesLicensesLinks   {
  private String _386522 = null;

  public HomeServicesLicensesLinks _386522(String _386522) {
    this._386522 = _386522;
    return this;
  }

  /**
   * Link to a license.
   * @return _386522
   **/
    public String get386522() {
    return _386522;
  }

  public void set386522(String _386522) {
    this._386522 = _386522;
  }


  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    HomeServicesLicensesLinks homeServicesLicensesLinks = (HomeServicesLicensesLinks) o;
    return Objects.equals(this._386522, homeServicesLicensesLinks._386522);
  }

  @Override
  public int hashCode() {
    return Objects.hash(_386522);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class HomeServicesLicensesLinks {\n");
    
    sb.append("    _386522: ").append(toIndentedString(_386522)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}
